css/style.css

```diff
@@ -1,9 +1,199 @@
-
- /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
+/* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
 
 body {
-    font-family: 'Roboto', sans-serif;
-    -webkit-font-smoothing: antialiased;
+  font-family: "Roboto", sans-serif;
+  -webkit-font-smoothing: antialiased;
+  margin: 160px;
+  font-size: 1em;
```

Comment:
It's best not to add specific styles like this to the `body` - it's too generic and you'll want to override those styles most of the time. Start adding your styles one level down: on `header`, `main` and `footer`

Reply:
Also, to achieve [this design](https://raw.githubusercontent.com/CodeYourFuture/HTML-CSS-Coursework-Week2/master/Karma%20-%20desktop%20design.png), we don't want to create all that white space around the page. We want the main background image to 'bleed' off the page. To achieve this, remove the margin from line 6.